.changeset/sour-weeks-wait.md

```diff
@@ -0,0 +1,29 @@
+---
+"@lynx-js/react-rsbuild-plugin": patch
+---
+
+Add `react-compiler-runtime` to `resolve.dedupe`.
+
+With this change you can setup [React Compiler](https://react.dev/learn/react-compiler) for ReactLynx by `pluginBabel`:
```

Comment:
We may also want to have a guide on https://lynxjs.org/react for using React Compiler

Reply:
Sure, I will update the docs later!